qosf.org/learn_quantum.md

```diff
@@ -208,16 +199,8 @@ Question and answer site for quantum computing.
 #### [Quantum Information and Quantum Computer Scientists of the World Unite](https://www.facebook.com/groups/qinfo.scientists.unite/)
 Facebook group for quantum research discussion.
 
-#### [Rigetti Community](http://slack.rigetti.com/)
```

Comment:
As the current link is not working, requested new slack link here - https://x.com/rigetti/status/1038161517307297792